core/src/main/java/io/camunda/migrator/persistence/IdKeyMapper.java

```diff
@@ -21,6 +21,7 @@ enum TYPE {
     HISTORY_USER_TASK,
     HISTORY_FLOW_NODE,
     HISTORY_DECISION_INSTANCE,
+    HISTORY_DECISION_DEFINITION,
```

Comment:
❓ I adjusted this with the assumption that the plan is to enable decision migration, is this correct? Otherwise I would remove this and the decision defintion migration method in the historyMigrator.

Reply:
Resolved as per [slack convo](https://camunda.slack.com/archives/C036ZURJFMX/p1753092921671669)